tests/verifyRemoteInstances_test.js

```diff
@@ -181,3 +187,23 @@ Scenario(
     await pmmInventoryPage.checkAgentOtherDetailsSection('listen_port:', 'listen_port: 42100', serviceName, serviceId);
   },
 );
+*/
+// add postgresPgStatMonitor after fix https://jira.percona.com/browse/PMM-8054
```

Comment:
Do we really need to skip all the steps only because the agent status of `running` is not displayed? 

I'd rather just add a check at the steps which are looking for pgStatMonitor agent status after adding, that way we still cover 50% of user flow.